simplemma/langdetect.py

```diff
@@ -30,15 +30,21 @@ def prepare_text(text: str, splitting_regex: Pattern[str] = SPLIT_INPUT) -> List
     return [item[0] for item in counter.most_common(1000)]
 
 
-def in_target_language(text: str, lang: Optional[Tuple[str]] = None) -> float:
+def in_target_language(
+    text: str,
+    lang: Optional[Tuple[str]] = None,
+    dictionaryFactory: DictionaryFactory = DictionaryFactory(),
```

Comment:
"Names should be lowercase, with words separated by underscores as necessary to improve readability" as of PEP 8, so _dictionary_factory_ would be more pythonic.

Reply:
Solved!